### ISSUE TITLE
Add more responses to each action

### ISSUE THREAD

**Issue by rakki-18** [adityasohoni/IEEE_Chatbot_v2#14]
Currently, for a particular action, the model replies with the same response. 
Ex: if action is ```utter_iamabot``` the response is always ```I am a bot, powered by Rasa.```
It would be nice to have different responses to the same action instead of repeating the same response every time.

**Comment by adityasohoni** [comment]
Yes, this is something that will make the bot more human-like in conversations. 
How do you plan to achieve this? Will you be making a custom action?

**Comment by spectre900** [comment]
**We can just add more examples so that it picks one of them randomly.**

<img src="https://user-images.githubusercontent.com/31584170/103072251-3db78300-45eb-11eb-8a84-322522274970.png" height=300 width=550>

**Comment by adityasohoni** [comment]
Alright you can do this👍
